components/esp_wifi_remote/scripts/copyright_header.h

```diff
@@ -0,0 +1,5 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
```

Comment:
How it is handled when year changes?

Reply:
I kept the workflow, but added a description in the checker what to do and which file to update if it fails.